### ISSUE TITLE
Suggestion for /is restart for other players

### ISSUE THREAD

**Issue by PretoAlbino** [KaiKikuchi/GriefPreventionPlus-SkyBlock#17]
If there's possible, when a player has OP, can use /is restart in other player ilsands, thanks

**Comment by KaiKikuchi** [comment]
Maybe something like `/isadmin reset [playername]`

**Comment by PretoAlbino** [comment]
Yes, other things with isadmin is good too, like /isadmin [setbiome | purge(to kill all mobs in island)], if theres possible too, a command to make other player the leader of the island, like /is makeleader.
